src/fastcs/attributes.py

```diff
@@ -186,21 +189,26 @@ async def process(self, value: T) -> None:
         await self.update_display_without_process(value)
 
     async def process_without_display_update(self, value: T) -> None:
-        if self._process_callback is not None:
-            await self._process_callback(self._datatype.validate(value))
+        if self._process_callbacks:
+            await asyncio.gather(*[cb(value) for cb in self._process_callbacks])
 
     async def update_display_without_process(self, value: T) -> None:
-        if self._write_display_callback is not None:
-            await self._write_display_callback(self._datatype.validate(value))
+        self._datatype.validate(value)
+        if self._write_display_callbacks:
+            await asyncio.gather(*[cb(value) for cb in self._write_display_callbacks])
 
-    def set_process_callback(self, callback: AttrCallback[T] | None) -> None:
-        self._process_callback = callback
+    def add_process_callback(self, callback: AttrCallback[T]) -> None:
+        if self._process_callbacks is None:
+            self._process_callbacks = []
+        self._process_callbacks.append(callback)
 
     def has_process_callback(self) -> bool:
-        return self._process_callback is not None
+        return self._process_callbacks is not None and len(self._process_callbacks) != 0
```

Comment:
```suggestion
        return self._process_callbacks is not None and self._process_callbacks
```
empty lists are falsy